### PR TITLE
Removed unused variables, general code clean up

### PR DESCRIPTION
**Pull request by smarthi**

### CODE REVIEW

File: src/main/java/org/apache/pirk/responder/wideskies/spark/ComputeResponse.java [hunk 1]
```diff
@@ -316,23 +311,15 @@ public void performQuery() throws Exception
...
    */
   public void performQuery(JavaRDD<MapWritable> inputRDD) throws PIRException
   {
     logger.info("Performing query: ");
 
-    // If we are using distributed exp tables -- Create the expTable file in hdfs for this query, if it doesn't exist
-    if ((queryInfo.getUseHDFSExpLookupTable() || useHDFSLookupTable) && query.getExpFileBasedLookup().isEmpty())
-    {
-      // <queryHash, <<power>,<element^power mod N^2>>
-      JavaPairRDD<Integer,Iterable<Tuple2<Integer,BigInteger>>> expCalculations = ComputeExpLookupTable.computeExpTable(sc, fs, bVars, query, queryInput,
```
Comment (tellison) [inline src/main/java/org/apache/pirk/responder/wideskies/spark/ComputeResponse.java]: Is this removal safe?  Although the return value is ignored, the compute method has the side-effect of creating the exponent look-up table in HDFS, and storing it in the Query.

File: src/main/java/org/apache/pirk/schema/query/QuerySchemaLoader.java [hunk 2]
```diff
@@ -296,7 +296,7 @@ private Document parseXMLDocument(InputStream stream) throws IOException, PIRExc
    * @param doc
    *          An XML document specifying names upon which we will filter the query.
    * @return The set of names upon which we will filter the query.
-   * @throws PIRException
+   * @throws PIRException -
```
Comment (tellison) [inline src/main/java/org/apache/pirk/schema/query/QuerySchemaLoader.java]: I'd rather keep the warnings than read this throws comment :-)

File: src/main/java/org/apache/pirk/responder/wideskies/spark/ComputeResponse.java (hunk 1 above)
Comment (smarthi) [inline src/main/java/org/apache/pirk/responder/wideskies/spark/ComputeResponse.java]: It seems 'Safely Unsafe', will revert this.

**Comment by smarthi** [comment]
@tellison could u review again? Updated the PR addressing ur previous comments.

**Comment by ellisonanne** [comment]
+1

**Pull request closed without merging**